woof-code/packages-templates/firefox_FIXUPHACK

```diff
@@ -11,7 +11,6 @@ defaultPref("browser.uidensity", 1);
 
 // render website text while fetching images
 defaultPref("nglayout.initialpaint.delay", 0);
-defaultPref("layers.progressive-paint", true);
```

Comment:
So _that_ is the culprit.

When I find time I'll do a new slacko 8 build from from `c201-v2`